src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -905,9 +1017,29 @@ public final class AddressList implements ApiMessage {
     return fieldMap;
   }
 
+  @Override
+  public String getFieldStringValue(String fieldName) {
+    if (fieldName.equals("id")) {
+      return String.valueOf(id);
+    }
+    if (fieldName.equals("items")) {
+      return String.valueOf(items);
```

Comment:
I don't think `String.valueOf` will return anything useful for lists... (you can solve this problem later, just add a TODO)

Reply:
Done.